react/package.json

```diff
@@ -29,27 +31,52 @@
   },
   "devDependencies": {
     "@testing-library/react-hooks": "^2.0.1",
+    "@vtex/prettier-config": "^0.3.1",
     "@vtex/test-tools": "^1.2.0",
     "@vtex/tsconfig": "^0.5.0",
     "apollo-cache-inmemory": "^1.2.2",
     "apollo-client": "^2.5.1",
     "apollo-client-preset": "^1.0.8",
     "babel-jest": "^24.1.0",
     "babel-preset-react-app": "^7.0.1",
-    "eslint": "^5.15.3",
-    "eslint-config-vtex-react": "^4.0.0",
+    "eslint": "^7.4.0",
+    "eslint-config-vtex": "^12.7.0",
+    "eslint-config-vtex-react": "^6.7.0",
     "graphql": "^0.13.2",
     "graphql-tag": "^2.8.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.1.0",
     "jest-dom": "^3.1.2",
     "jest-transform-graphql": "^2.1.0",
-    "prettier": "^1.16.4",
+    "prettier": "^2.0.5",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
     "react-test-renderer": "^16.7.0",
     "regenerator-runtime": "^0.13.1",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "vtex.blog-interfaces": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.blog-interfaces@0.2.0/public/@types/vtex.blog-interfaces",
+    "vtex.breadcrumb": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.breadcrumb@1.9.1/public/@types/vtex.breadcrumb",
+    "vtex.css-handles": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.css-handles@0.4.4/public/@types/vtex.css-handles",
+    "vtex.device-detector": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.device-detector@0.2.5/public/@types/vtex.device-detector",
+    "vtex.flex-layout": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.flex-layout@0.15.1/public/@types/vtex.flex-layout",
+    "vtex.format-currency": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.format-currency@0.2.0/public/@types/vtex.format-currency",
+    "vtex.native-types": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.native-types@0.7.4/public/@types/vtex.native-types",
+    "vtex.pixel-manager": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.pixel-manager@1.4.0/public/@types/vtex.pixel-manager",
+    "vtex.product-list-context": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.product-list-context@0.3.0/public/@types/vtex.product-list-context",
+    "vtex.product-summary": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.product-summary@2.65.0/public/@types/vtex.product-summary",
+    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.126.4-beta.1/public/@types/vtex.render-runtime",
+    "vtex.responsive-values": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.responsive-values@0.3.0/public/_types/react",
+    "vtex.rich-text": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.rich-text@0.11.2/public/@types/vtex.rich-text",
+    "vtex.search-page-context": "https://icarosearch--storecomponents.myvtex.com/_v/private/typings/linked/v1/vtex.search-page-context@0.1.0+build1607092293/public/@types/vtex.search-page-context",
```

Comment:
Update dependency when [search-page-context#8](https://github.com/vtex-apps/search-page-context/pull/8) is merged